sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -57,13 +57,13 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 79c28d0f0713f58358d5080653d95803fd131749
-  # Merge: 25e0b603 45d76b78
-  # Author: aarongreig <aaron.greig@codeplay.com>
-  # Date: Fri Jan 12 16:14:44 2024 +0000
-  #     Merge pull request #1186 from hdelan/device-global-hip
-  #     [HIP] Add support for global variable read write
-  set(UNIFIED_RUNTIME_TAG 79c28d0f0713f58358d5080653d95803fd131749)
+  # commit 5d58871ccce3dd605bdc4245819965cc77441946
+  # Merge: 8007b222 0ebaca6e
+  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
+  # Date:   Wed Jan 17 12:35:19 2024 +0000
+  #     Merge pull request #1245 from nrspruit/enable_relaxed_alloc
+  #     [L0] Only Override max allocation limits given env
+  set(UNIFIED_RUNTIME_TAG 5d58871ccce3dd605bdc4245819965cc77441946)
```

Comment:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/kbenzie/unified-runtime.git")
  set(UNIFIED_RUNTIME_TAG 6036025e127e1460e188861db0ce81de659ee7e6)
```

Reply:
@kbenzie , I updated this right now so we can see if that fixes the issue, thanks!

Reply:
sorry, the correct commit is eba3905b1cb3e821401f6b9268358b2684be697d

Reply:
I Just updated to your latest commit, thanks!